skflow/estimators/base.py

```diff
@@ -62,12 +62,18 @@ class TensorFlowEstimator(BaseEstimator):
         early_stopping_rounds: Activates early stopping if this is not None.
             Loss needs to decrease at least every every <early_stopping_rounds>
             round(s) to continue training. (default: None)
-
+        max_to_keep: The maximum number of recent checkpoint files to keep.
+            As new files are created, older files are deleted.
+            If None or 0, all checkpoint files are kept.
+            Defaults to 5 (that is, the 5 most recent checkpoint files are kept.)
+        keep_checkpoint_every_n_hours: Number of hours between each checkpoint
+            to be saved. The default value of 10,000 hours effectively disables the feature.
     """
 
     def __init__(self, model_fn, n_classes, tf_master="", batch_size=32, steps=50, optimizer="SGD",
                  learning_rate=0.1, tf_random_seed=42, continue_training=False,
-                 num_cores=4, verbose=1, early_stopping_rounds=None):
+                 num_cores=4, verbose=1, early_stopping_rounds=None,
```

Comment:
Options added for other estimators. Yeah we'll need to think about this soon. One idea is to accept dict type parameters for each unique functionality, e.g. use a parameter called `saver_options` that accepts `{"max_to_keep": 5, "keep_checkpoint_every_n_hours": 2}`, or consolidates some parameters once more parameters are coming in the future. 

Fortunately it's actually not that bad, most of them have default values and very few of them are tunable.